spec/requests/projects/create_spec.rb

```diff
@@ -4,13 +4,26 @@
 
 RSpec.describe 'ProjectController#create' do
   let(:path) { '/projects' }
-  let(:headers) { {} }
+  let(:user) { create(:user) }
+  let(:headers) { { 'Accept' => 'application/json', 'Content-Type' => 'application/json' } }
+  let(:auth_headers) { Devise::JWT::TestHelpers.auth_headers(headers, user) }
 
   before :example do
-    post(path, params: params, headers: headers)
+    post(path, params: params, headers: auth_headers, as: :json)
   end
 
   context 'with invalid credentials' do
+    let(:auth_headers) { headers.tap {|h| h['Authorization'] = ''} }
```

Comment:
we are missing spaces around the block, it also applies to the other specs in areas where you are tapping headers

```suggestion
    let(:auth_headers) { headers.tap { |h| h['Authorization'] = '' } }
```